spec/features/viewing_peeps_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'pg'
+
+feature 'Viewing peeps' do 
+  scenario 'The /peeps page displays all peeps' do
+    add_row_to_test_database
+
+    visit '/peeps'
+    
+    expect(page).to have_content "This is a peep!"
+  end
+
+  scenario 'The user can see the date the peep was posted' do 
+    add_row_to_test_database
```

Comment:
Minor: Using a helper method here helps to reduce duplication. One drawback of helper methods though is that they can make the test harder to follow as a reader of the test doesn't know at first glance what kind of data is being added to the database.

A compromise could be to make your helper method take an argument, e.g. the text of the peep so that you can call it like this `add_row_to_test_database("some text")`. That would make the link between line 17 and line 13 clearer.